ui/i18n/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -30,6 +30,8 @@
   <string name="swipe_left">左滑</string>
   <string name="enable_swipeactions">启用此屏幕的滑动操作 </string>
   <string name="change_setting">更改</string>
+  <string name="do_not_show_remove_inbox_prompt">不再询问.</string>
```

Comment:
Please don't touch the translation files. They are handled only on Transifex

Reply:
Got it, I'll remove it.